TP_Model/OSL_TFA_L_Ankle_Modified_cvt3.xml

```diff
@@ -0,0 +1,688 @@
+        <site name="psoas_l_psoas_l-P4" pos="0.0016 -0.0507 -0.0038"/>
+        <site name="psoas_l_psoas_l-P5" pos="-0.0188 -0.0597 -0.0104"/>
+        <site name="quad_fem_l_quad_fem_l-P2" pos="-0.0381 -0.0359 -0.0366"/>
+        <site name="gem_l_gem_l-P2" pos="-0.0142 -0.0033 -0.0443"/>
+        <site name="peri_l_peri_l-P3" pos="-0.0148 -0.0036 -0.0437"/>
+        <site name="rect_fem_l_rect_fem_l-P2" pos="0.0334 -0.403 -0.0019"/>
+        <site name="vas_int_l_vas_int_l-P1" pos="0.029 -0.1924 -0.031"/>
+        <site name="vas_int_l_vas_int_l-P2" pos="0.0335 -0.2084 -0.0285"/>
+        <site name="med_gas_l_med_gas_l-P1" pos="-0.019 -0.3929 0.0235"/>
+        <body name="tibia_l">
+          <inertial pos="0 -0.1867 0" quat="0.5 0.5 -0.5 0.5" mass="3.7075" diaginertia="0.0511 0.0504 0.0051"/>
+          <joint name="knee_l_translation1" pos="0 0 0" axis="1 0 0" type="slide" range="-0.005574 0.00411" user="-0.00363925"/>
+          <joint name="knee_l_translation2" pos="0 0 0" axis="0 1 0" type="slide" range="-0.4226 -0.3953" user="-0.395713"/>
+          <joint name="knee_angle_l" pos="0 0 0" axis="0 0 1" limited="false" range="-2.094 0.1745"/>
+          <geom name="tibia_l_geom_1" type="mesh" rgba="1 1 1 1" mesh="tibia_l_geom_1_tibia_l"/>
+          <geom name="tibia_l_geom_2" type="mesh" rgba="1 1 1 1" mesh="tibia_l_geom_2_l_fibula"/>
+          <site name="bifemlh_l_bifemlh_l-P2" pos="-0.0301 -0.036 -0.02943"/>
+          <site name="bifemlh_l_bifemlh_l-P3" pos="-0.0234 -0.0563 -0.0343"/>
+          <site name="bifemsh_l_bifemsh_l-P2" pos="-0.0301 -0.036 -0.02943"/>
+          <site name="bifemsh_l_bifemsh_l-P3" pos="-0.0234 -0.0563 -0.0343"/>
+          <site name="sar_l_sar_l-P3" pos="-0.0056 -0.0419 0.0399"/>
+          <site name="sar_l_sar_l-P4" pos="0.006 -0.0589 0.0383"/>
+          <site name="sar_l_sar_l-P5" pos="0.0243 -0.084 0.0252"/>
+          <body name="AnkleAssembly" pos="-0.04 -0.124 -0.05">
+            <inertial pos="0.04136 -0.24932 0.04006" quat="0.341335 0.435693 -0.739061 0.383994" mass="1.71" diaginertia="0.00220091 0.00207699 0.0012951"/>
+            <geom name="AnkleAssembly_geom_1" type="mesh" rgba="0.87 0.56 0.34 1" mesh="AnkleAssembly_geom_1_AnkleAssembly"/>
+            <body name="Foot">
+              <inertial pos="0.06215 -0.35314 0.04496" quat="0.130389 0.68448 0.215635 0.684096" mass="0.76" diaginertia="0.00184373 0.001789 0.001"/>
+              <joint name="yR_FootToAnkleAssembly" pos="0 0 0" axis="-1 0 0" range="-0.8727 0.5236"/>
+              <joint name="zR_FootToAnkleAssembly" pos="0 0 0" axis="0.105003 0.174005 0.979131" range="-0.45 0.6"/>
+              <geom name="Foot_geom_1" type="mesh" rgba="0.301961 0.301961 0.301961 1" mesh="Foot_geom_1_Foot_Amp"/>
+            </body>
+          </body>
+          <body name="rect_fem_l_rect_fem_l-P3" pos="0 0 -0.001401">
+            <joint name="rect_fem_l_rect_fem_l-P3_x" pos="0 0 0" axis="1 0 0" type="slide" range="0.01558 0.0733"/>
```

Comment:
movable muscle attachment sites are tricky to troubleshoot. Better to start with fixed points or to convert them to fixed points